[2021-12-28 13:42:02,307] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_tipperary.scrap_data scheduled__2021-12-27T10:50:00+00:00 [queued]>
[2021-12-28 13:42:02,397] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_tipperary.scrap_data scheduled__2021-12-27T10:50:00+00:00 [queued]>
[2021-12-28 13:42:02,423] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:42:02,429] {taskinstance.py:1242} INFO - Starting attempt 3 of 4
[2021-12-28 13:42:02,433] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:42:02,504] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-27 10:50:00+00:00
[2021-12-28 13:42:02,537] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_tipperary', 'scrap_data', 'scheduled__2021-12-27T10:50:00+00:00', '--job-id', '5100', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_tipperary.py', '--cfg-path', '/tmp/tmpjbzw690r', '--error-file', '/tmp/tmpach6si9x']
[2021-12-28 13:42:02,539] {standard_task_runner.py:77} INFO - Job 5100: Subtask scrap_data
[2021-12-28 13:42:02,538] {standard_task_runner.py:52} INFO - Started process 53138 to run task
[2021-12-28 13:42:02,795] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_tipperary.scrap_data scheduled__2021-12-27T10:50:00+00:00 [running]> on host 0c125a5bba95
[2021-12-28 13:42:03,042] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_tipperary
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T10:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T10:50:00+00:00
[2021-12-28 13:42:03,073] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 13:42:03,077] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py tipperary
[2021-12-28 13:42:07,079] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 13:42:07,249] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 13:42:07,250] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 13:42:07,251] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 13:42:07,252] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 13:42:07,253] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 13:42:07,326] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 13:42:07,327] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 13:42:07,329] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 13:42:07,330] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 13:42:07,331] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 13:42:07,332] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 13:42:07,333] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 13:42:07,334] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 13:42:07,335] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 13:42:07,335] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 13:42:07,336] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 13:42:07,338] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 13:42:07,339] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 13:42:07,340] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 13:42:07,341] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 13:42:07,342] {spark_submit.py:514} INFO - files                   null
[2021-12-28 13:42:07,343] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 13:42:07,344] {spark_submit.py:514} INFO - archives                null
[2021-12-28 13:42:07,345] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 13:42:07,346] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:42:07,346] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-28 13:42:07,347] {spark_submit.py:514} INFO - childArgs               [tipperary]
[2021-12-28 13:42:07,348] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 13:42:07,349] {spark_submit.py:514} INFO - packages                null
[2021-12-28 13:42:07,350] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 13:42:07,351] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 13:42:07,352] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 13:42:07,353] {spark_submit.py:514} INFO - 
[2021-12-28 13:42:07,354] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 13:42:07,355] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 13:42:07,356] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:42:07,356] {spark_submit.py:514} INFO - 
[2021-12-28 13:42:07,357] {spark_submit.py:514} INFO - 
[2021-12-28 13:42:07,886] {spark_submit.py:514} INFO - Main class:
[2021-12-28 13:42:07,887] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 13:42:07,888] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 13:42:07,889] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:42:07,890] {spark_submit.py:514} INFO - null
[2021-12-28 13:42:07,891] {spark_submit.py:514} INFO - tipperary
[2021-12-28 13:42:07,892] {spark_submit.py:514} INFO - --verbose
[2021-12-28 13:42:07,893] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 13:42:07,894] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-28 13:42:07,895] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:42:07,896] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 13:42:07,896] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 13:42:07,897] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 13:42:07,898] {spark_submit.py:514} INFO - 
[2021-12-28 13:42:07,899] {spark_submit.py:514} INFO - 
[2021-12-28 13:42:07,900] {spark_submit.py:514} INFO - 
[2021-12-28 13:42:09,170] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 13:42:09,191] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 13:42:09,425] {spark_submit.py:514} INFO - 21/12/28 13:42:09 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 13:42:09,671] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO ResourceUtils: ==============================================================
[2021-12-28 13:42:09,673] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 13:42:09,674] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO ResourceUtils: ==============================================================
[2021-12-28 13:42:09,675] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-28 13:42:09,741] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 13:42:09,767] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 13:42:09,769] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 13:42:09,904] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 13:42:09,906] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 13:42:09,907] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 13:42:09,908] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 13:42:09,909] {spark_submit.py:514} INFO - 21/12/28 13:42:09 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 13:42:10,381] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO Utils: Successfully started service 'sparkDriver' on port 35809.
[2021-12-28 13:42:10,428] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 13:42:10,483] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 13:42:10,522] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 13:42:10,524] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 13:42:10,539] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 13:42:10,584] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-b007606d-2c65-49f8-a9fc-12c03f80aa34
[2021-12-28 13:42:10,617] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 13:42:10,648] {spark_submit.py:514} INFO - 21/12/28 13:42:10 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 13:42:11,096] {spark_submit.py:514} INFO - 21/12/28 13:42:11 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-28 13:42:11,097] {spark_submit.py:514} INFO - 21/12/28 13:42:11 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-28 13:42:11,098] {spark_submit.py:514} INFO - 21/12/28 13:42:11 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-28 13:42:11,099] {spark_submit.py:514} INFO - 21/12/28 13:42:11 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2021-12-28 13:42:11,100] {spark_submit.py:514} INFO - 21/12/28 13:42:11 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2021-12-28 13:42:11,122] {spark_submit.py:514} INFO - 21/12/28 13:42:11 INFO Utils: Successfully started service 'SparkUI' on port 4045.
[2021-12-28 13:42:11,226] {spark_submit.py:514} INFO - 21/12/28 13:42:11 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4045
[2021-12-28 13:42:11,682] {spark_submit.py:514} INFO - 21/12/28 13:42:11 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:42:11,821] {spark_submit.py:514} INFO - 21/12/28 13:42:11 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:42:11,823] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:42:11,825] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:42:11,826] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:42:11,827] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:42:11,827] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:42:11,828] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:42:11,829] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:42:11,830] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:42:11,831] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:42:11,832] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:42:11,833] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:42:11,834] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:42:11,834] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:42:11,835] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:42:11,836] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:42:11,837] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:42:11,837] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:42:11,838] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:42:11,839] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:42:11,840] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:42:11,840] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:42:11,841] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:42:11,842] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:42:11,842] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:42:11,843] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:42:11,844] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:42:11,845] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:42:11,845] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:42:11,846] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:42:11,847] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:42:11,848] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:42:11,848] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:42:11,850] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:42:11,851] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:42:11,851] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:42:11,852] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:42:11,853] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:42:11,853] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:42:11,854] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:42:11,855] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:42:11,855] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:42:11,856] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:42:11,857] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:42:11,858] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:42:11,859] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:42:11,860] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:42:11,860] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:42:11,861] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:42:11,862] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:42:11,863] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:42:11,863] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:42:11,864] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:42:11,865] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:42:11,865] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:42:11,866] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:42:11,867] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:42:11,868] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:42:31,681] {spark_submit.py:514} INFO - 21/12/28 13:42:31 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:42:31,720] {spark_submit.py:514} INFO - 21/12/28 13:42:31 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:42:31,721] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:42:31,722] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:42:31,723] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:42:31,724] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:42:31,727] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:42:31,728] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:42:31,730] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:42:31,731] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:42:31,732] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:42:31,733] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:42:31,734] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:42:31,735] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:42:31,736] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:42:31,737] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:42:31,738] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:42:31,739] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:42:31,740] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:42:31,740] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:42:31,741] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:42:31,741] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:42:31,742] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:42:31,743] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:42:31,743] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:42:31,744] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:42:31,745] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:42:31,746] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:42:31,746] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:42:31,747] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:42:31,748] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:42:31,749] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:42:31,750] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:42:31,750] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:42:31,751] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:42:31,752] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:42:31,752] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:42:31,754] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:42:31,754] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:42:31,755] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:42:31,755] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:42:31,756] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:42:31,757] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:42:31,757] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:42:31,758] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:42:31,759] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:42:31,759] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:42:31,760] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:42:31,761] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:42:31,762] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:42:31,763] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:42:31,763] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:42:31,764] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:42:31,766] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:42:31,767] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:42:31,768] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:42:31,768] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:42:31,769] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:42:31,770] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:42:51,647] {spark_submit.py:514} INFO - 21/12/28 13:42:51 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:42:51,667] {spark_submit.py:514} INFO - 21/12/28 13:42:51 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:42:51,668] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:42:51,669] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:42:51,670] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:42:51,671] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:42:51,672] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:42:51,673] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:42:51,674] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:42:51,675] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:42:51,676] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:42:51,677] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:42:51,678] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:42:51,678] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:42:51,679] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:42:51,681] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:42:51,682] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:42:51,683] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:42:51,684] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:42:51,684] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:42:51,685] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:42:51,686] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:42:51,687] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:42:51,687] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:42:51,688] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:42:51,689] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:42:51,689] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:42:51,690] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:42:51,690] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:42:51,691] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:42:51,692] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:42:51,692] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:42:51,693] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:42:51,694] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:42:51,695] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:42:51,695] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:42:51,696] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:42:51,697] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:42:51,698] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:42:51,699] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:42:51,700] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:42:51,700] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:42:51,701] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:42:51,702] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:42:51,702] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:42:51,703] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:42:51,704] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:42:51,705] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:42:51,705] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:42:51,706] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:42:51,707] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:42:51,707] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:42:51,708] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:42:51,709] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:42:51,709] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:42:51,710] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:42:51,712] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:42:51,712] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:42:51,713] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:43:11,764] {spark_submit.py:514} INFO - 21/12/28 13:43:11 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 13:43:11,796] {spark_submit.py:514} INFO - 21/12/28 13:43:11 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 13:43:11,958] {spark_submit.py:514} INFO - 21/12/28 13:43:11 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4045
[2021-12-28 13:43:12,011] {spark_submit.py:514} INFO - 21/12/28 13:43:11 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 38953.
[2021-12-28 13:43:12,023] {spark_submit.py:514} INFO - 21/12/28 13:43:11 INFO NettyBlockTransferService: Server created on 0c125a5bba95:38953
[2021-12-28 13:43:12,025] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 13:43:12,033] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 13:43:12,057] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 13:43:12,077] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 38953, None)
[2021-12-28 13:43:12,161] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:38953 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 38953, None)
[2021-12-28 13:43:12,184] {spark_submit.py:514} INFO - 21/12/28 13:43:12 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 13:43:12,186] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 38953, None)
[2021-12-28 13:43:12,194] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 38953, None)
[2021-12-28 13:43:12,287] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 13:43:12,561] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO MemoryStore: MemoryStore cleared
[2021-12-28 13:43:12,569] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO BlockManager: BlockManager stopped
[2021-12-28 13:43:12,633] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 13:43:12,679] {spark_submit.py:514} INFO - 21/12/28 13:43:12 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 13:43:15,268] {spark_submit.py:514} INFO - 21/12/28 13:43:13 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 13:43:15,335] {spark_submit.py:514} INFO - 21/12/28 13:43:14 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 13:43:15,363] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:43:15,373] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:43:15,377] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:43:15,382] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:43:15,390] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:43:15,394] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:43:15,404] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:43:15,405] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:43:15,413] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:43:15,433] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:43:15,458] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:43:15,461] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:43:15,470] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:43:15,510] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:43:15,567] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:43:15,606] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:43:15,637] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:43:15,704] {spark_submit.py:514} INFO - 21/12/28 13:43:14 INFO SparkContext: SparkContext already stopped.
[2021-12-28 13:43:15,710] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 13:43:15,746] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-28 13:43:15,756] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-28 13:43:15,773] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 13:43:15,842] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 13:43:15,879] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 13:43:15,903] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 13:43:15,908] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 13:43:15,911] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 13:43:15,924] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 13:43:15,935] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 13:43:15,943] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:43:15,969] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:43:15,983] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:43:16,003] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:43:16,018] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:43:16,159] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:43:16,172] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:43:16,176] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:43:16,183] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:43:16,192] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:43:16,201] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:43:16,261] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:43:16,309] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:43:16,330] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:43:16,345] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:43:16,370] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:43:16,386] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:43:16,431] {spark_submit.py:514} INFO - 
[2021-12-28 13:43:18,779] {spark_submit.py:514} INFO - 21/12/28 13:43:18 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 13:43:18,928] {spark_submit.py:514} INFO - 21/12/28 13:43:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-c52dc9de-9368-4dea-a2fe-d95c5f44faff
[2021-12-28 13:43:19,003] {spark_submit.py:514} INFO - 21/12/28 13:43:19 INFO ShutdownHookManager: Deleting directory /tmp/spark-59169597-d59a-4e02-8231-dfa6be536373
[2021-12-28 13:43:20,691] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py tipperary. Error code is: 1.
[2021-12-28 13:43:20,741] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_tipperary, task_id=scrap_data, execution_date=20211227T105000, start_date=20211228T134202, end_date=20211228T134320
[2021-12-28 13:43:20,854] {standard_task_runner.py:91} ERROR - Failed to execute job 5100 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py tipperary. Error code is: 1.
[2021-12-28 13:43:20,987] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 13:43:21,690] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
